local.env~

```diff
@@ -0,0 +1,30 @@
+# Environment Configuration #
+#############################
+RAILS_ENV=development
+
+# General Configuration #
+#########################
+LANG=en_US.UTF-8
+RAILS_SERVE_STATIC_FILES=
+SECRET_KEY_BASE=e1afcadc83e3e925c4af435146fe72c7b8c5610a03602e46fc54eccdffa16b521dcc73129901d6a8cc4dfba3daec65370eebdb8f822a9774a89c5273f92a5f38
+
+# MySQL Configruation #
+#######################
+DB_HOSTNAME=mysql
+DB_PORT=3306
+DB_USERNAME=suyc
+DB_PASSWORD=suyc
+DB_NAME=suyc
+
+# 3rd party API integrations #
+##############################
+MAPBOX_API_KEY=
+LOGGLY_TOKEN=
+ALGOLIA_APP_ID=
+ALGOLIA_API_KEY=
+
+# MLab's Big Query Configuration #
+##################################
+MLAB_BIGQUERY_EMAIL=
+MLAB_BIGQUERY_PRIVATE_KEY=
+MLAB_BIGQUERY_PRIVATE_KEY_PASSPHRASE=
```

Comment:
Don't commit this file. Looks like it wasn't ignored by `.gitignore` because it's named `local.env~`

Reply:
Weird, looks like that was auto-generated by emacs in addition to the normal local.env.